self-hosting/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18.16-alpine as builder
+FROM node:18.16.1-alpine as builder
```

Comment:
Do you know if we need this?

Reply:
It's [best practice](https://hackernoon.com/docker-images-name-vs-tag-vs-digest) if you want images to work on machines other than yours. By pinning the minor version of the node image, it can still get security updates (vs. pinning manifest), but is guaranteed to work with the version of node Omnivore was tested with.

So for example, I can be fairly confident that we'll always get this
```bash
/app # node --version
v18.16.1
```
whereas if you only use the `node:18.16-alpine` tag, you can end up with `v18.6.2` or later (although it appears to [get updated every 5-6 months](https://hub.docker.com/layers/library/node/18.16-alpine/images/sha256-055cfc51875a99402319113cb69404f8cd5086e8f5b128bee71126c04990a97a?context=explore).)

It's your show, so understand if only specifying major/minor version is preferred.